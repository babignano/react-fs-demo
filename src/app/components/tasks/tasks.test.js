import React from 'react';
import Tasks from './tasks';
import renderer from 'react-test-renderer';

describe('Tasks', () => {
    test('should match snapshot', () => {
        const component = renderer.create(<Tasks />);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
