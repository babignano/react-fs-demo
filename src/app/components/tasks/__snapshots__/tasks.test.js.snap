// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tasks should match snapshot 1`] = `
<h1>
  Tasks
</h1>
`;
